src/main/java/com/ptrader/connector/kraken/KrakenApiClient.java

```diff
@@ -296,72 +140,75 @@ private LastIdExtractedResult extractLastId(String response) {
         params.put("pair", pair);
         params.put("since", String.valueOf(sincelastId));
 
-        return callPublicWithLastId(KrakenApiMethod.RECENT_SPREADS, RecentSpreadResult.class, params);
+        return this.krakenApiClientInternal.callPublicWithLastId(KrakenApiMethod.RECENT_SPREADS, RecentSpreadResult.class, params);
     }
 
+    /* ***************************************************************************************************************** */
+    /* ************ Private calls ************************************************************************************** */
+    /* ***************************************************************************************************************** */
+
```

Comment:
We could `<editor-fold>` to define code regions in intellJ, unless you don't like having IDE specific features in the code base, what do you think?